lib/Model.php

```diff
@@ -426,7 +426,7 @@ public function __set($name, $value)
 		foreach (static::$delegate as &$item)
 		{
 			if (($delegated_name = $this->is_delegated($name,$item)))
-				return $this->$item['to']->$delegated_name = $value;
+				return $this->{$item['to']}->$delegated_name = $value;
```

Comment:
We really should always be using the curly-brace wrappers whenever we nest references like this. I'd suggest changing this to:

```php
return $this->{$item['to']}->{$delegated_name} = $value;
```